src/settings.cpp

```diff
@@ -95,6 +95,7 @@ int n_log_bins {8000};
 int n_batches;
 int n_max_batches;
 int max_splits {1000};
+int max_tracks {std::numeric_limits<int>::max()};
```

Comment:
I realize that this is up to the user to know that writing all of the tracks generally isn't a good idea, but maybe we can choose a large number here that won't cause a user trying this feature for the first time to fill up their disk with particle tracks.

Reply:
Good point -- I'll set the default max to 1,000. For a coupled neutron-photon pincell problem, this gives me a file of about ~100 MB.